### ISSUE TITLE
ExprFurnaceSlot fails when using the (fuel|result) [slot] pattern

### ISSUE THREAD

**Issue by sovdeeth** [SkriptLang/Skript#4673]
### Skript/Server Version

```markdown
[14:27:39 INFO]: [Skript] Skript's aliases can be found here: https://github.com/SkriptLang/skript-aliases
[14:27:39 INFO]: [Skript] Skript's documentation can be found here: https://skriptlang.github.io/Skript
[14:27:39 INFO]: [Skript] Server Version: git-Paper-250 (MC: 1.18.2)
[14:27:39 INFO]: [Skript] Skript Version: 2.6.1
[14:27:39 INFO]: [Skript] Installed Skript Addons: None
[14:27:39 INFO]: [Skript] Installed dependencies: None
```

### Bug Description

The `(fuel|result) [slot]` expression errors when called. Logs are in the "errors or screenshots" section.
I believe this is because `setExpr()` is never called if the pattern is `(fuel|result) [slot]`. 

### Expected Behavior

This should broadcast the contents of the furnace's result slot.

### Steps to Reproduce

```
on smelt:
    broadcast result slot
```

```
on fuel burn:
    broadcast fuel slot
```

### Errors or Screenshots

https://paste.gg/p/anonymous/e4f34b2a867c4604b0f3f10961513671
(same error for `on fuel burn` as far as I can tell)

### Agreement

- [X] I have read the guidelines above and affirm I am following them with this report.

**Comment by APickledWalrus** [comment]
Probably fixed in https://github.com/SkriptLang/Skript/pull/4281

**Comment by AyhamAl-Ali** [comment]
> Probably fixed in #4281

Tested and it does :ok_hand:
